src/test/java/com/laborhelper/enterprise/LaborHelperApplicationTests.java

```diff
@@ -29,10 +28,47 @@ class WebLayerTest {
 
     @Autowired
     private MockMvc mockMvc;
+    private ILaborActionService laborActionService;
+    private LaborAction laborAction;
+
 
     @Test
     public void shouldReturnDefaultMessage() throws Exception {
         this.mockMvc.perform(get("/")).andDo(print()).andExpect(status().isOk())
                 .andExpect(content().string(containsString("Hello World")));
     }
+
+    /**
+    For Businesslogic
+    Given: a user has entered in a valid zip code of 45239
+    When: The Search Button is Pressed
+    Then: Display all results of that zip code
+     */
+
+
+    @Test
+    void whenuserentersvalidzipcode_returnresultsofzipcode()
+
+    {
+    ZipCodeisEntered();
+    SearchForResult();
+    ReturnzipCode();
```

Comment:
1) Method names  should be lowerCamelCase.
2) Use proper Given When Then syntax.